utils/docker/Dockerfile.centos.7

```diff
@@ -73,7 +73,7 @@ RUN yum -y install \
            python36-clustershell python36-paramiko             \
            python36-numpy python36-jira python3-pip            \
            fuse3-devel libipmctl-devel openmpi3-devel          \
-           hwloc-devel patchelf
+           hwloc-devel patchelf python36-tabulate
```

Comment:
do you need a similar change in other docker files if we want to run tests?

Reply:
It doesn't appear so.  The cart_logtest code is only used by the node_local_testing currently, and the other docker files are only used for builders where we don't run testing.

Reply:
What about when we start testing on other platforms, like Leap 15?  That one will be very soon.  I'm just tidying code up in preparation for reviews and landing.

Reply:
With the "node local testing" script on Docker?  That would be great.  If the package isn't installed it'll be apparent when that functionality is enabled, but I don't want to add packages to docker files that we might need in the future.